### ISSUE TITLE
Move qsat, qsat_water and cldfrc_fice to ESCOMP/atmospheric_physics

### ISSUE THREAD

**Issue by cacraigucar** [ESCOMP/CAM#963]
### Issue Type

Infrastructure Update

### Issue Description

The ZM physics scheme "use"s qsat, qsat_water and cldfrc_fice.  These need to be transferred over to ESCOMP/atmospheric_physics and CCPP'ized.  We will explore moving just these routines and not the entire module at this time.

### Will this change answers?

No

### Will you be implementing this yourself?

Yes

**Comment by swrneale** [comment]
@cacraigucar just to be clear this doesn't affect the latest tag running ZM out of the box pulling from atmos_physics repo. ?

**Comment by cacraigucar** [comment]
> @cacraigucar just to be clear this doesn't affect the latest tag running ZM out of the box pulling from atmos_physics repo. ?

That is correct.  There are a couple of "use" statements which are using these named routines (which CCPP does not allow).  This should be a "behind the scenes" change.  As long as the atmospheric_physics and CAM repos are synched, it will be fine.  So currently you should be using cam6_3_146.  If this issue becomes a PR, then once the two repos are updated, at some point you may want to merge them forward.

**Comment by gold2718** [comment]
> There are a couple of "use" statements which are using these named routines (which CCPP does not allow).

Why not? If you use the `dependencies` property of the metadata table header, I think they should be allowed.

**Comment by cacraigucar** [comment]
@gold2718 - Don't the routines need to be CCPP'ized (with metadata and such) in order to use the dependencies property?

**Comment by gold2718** [comment]
This keyword was originally requested by NOAA explicitly for shared code that is *not* a CCPP scheme so metadata is not really appropriate. The datatable script will tell you all of these files so they can be added to the build (depending on how you collect files for a build). I think adding the "USEd" filenames as dependencies here is entirely appropriate.

**Comment by cacraigucar** [comment]
Will use the "dependencies" keyword instead of doing partial conversions.
